server/controllers/users/deactivateUser.js

```diff
@@ -1,3 +1,16 @@
-exports.deactivateUser = (req, res) => {
-  res.send('Hi from deactivate file');
+const { users: { deactivateUser } } = require('../../database/queries');
+
+exports.deactivateUser = (req, res, next) => {
+  // Get the user id and username
+  // const { userId, username } = req.user;
```

Comment:
```suggestion
  // const { id, username } = req.user;
```
don't forget to be constents  see what @fadeomar make on the #86